### PR TITLE
Honister support.

### PR DESCRIPTION
**Pull request by kevin-vigor**
Changes required to build with honister.

### CODE REVIEW

**Comment by om26er** [comment]
LGTM, @bboozzoo can you take a look please ?

File: recipes-core/images/snapd-demo-image.bb [hunk 1]
```diff
@@ -16,4 +16,4 @@ LICENSE = "MIT"
 inherit core-image
 
 IMAGE_ROOTFS_SIZE ?= "819200"
-IMAGE_ROOTFS_EXTRA_SPACE_append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
+IMAGE_ROOTFS_EXTRA_SPACE:append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
```
Comment (bboozzoo) [inline recipes-core/images/snapd-demo-image.bb:19]: Nice, I wasn't aware of https://docs.yoctoproject.org/migration-guides/migration-3.4.html#override-syntax-changes

**Review by zyga — COMMENTED** [review]
Looks good :)

**Comment by bboozzoo** [comment]
Thank you!

**Pull request merged**